pytorch_lightning/profiler/profilers.py

```diff
@@ -117,37 +189,41 @@ def stop(self, action_name: str) -> None:
     def summary(self) -> str:
         return ""
 
+    def teardown(self) -> None:
+        pass
+
 
 class SimpleProfiler(BaseProfiler):
     """
     This profiler simply records the duration of actions (in seconds) and reports
     the mean duration of each action and the total time spent over the entire training run.
     """
 
-    def __init__(self, output_filename: Optional[str] = None, extended=True):
+    def __init__(
+        self,
+        dirpath: Optional[Union[str, Path]] = None,
+        filename: Optional[str] = None,
+        extended: bool = True,
+        output_filename: Optional[str] = None,
+    ) -> None:
         """
         Args:
-            output_filename: optionally save profile results to file instead of printing
-                to std out when training is finished.
+            dirpath: Directory path for the ``filename``. If ``dirpath`` is ``None`` but ``filename`` is present, the
+                ``trainer.log_dir`` (from :class:`~pytorch_lightning.loggers.tensorboard.TensorBoardLogger`)
```

Comment:
nit: this isn't specific to tensorboard right?

Reply:
I believe only TensorBoardLogger and CSVLogger define log_dir